infra/cifuzz/docker.py

```diff
@@ -123,6 +123,16 @@ def clusterfuzz_build(self):
     """The directory where builds from ClusterFuzz are stored."""
     return os.path.join(self.workspace, 'cifuzz-prev-build')
 
+  @property
+  def clusterfuzz_coverage(self):
+    """The directory where builds from ClusterFuzz are stored."""
+    return os.path.join(self.workspace, 'cifuzz-prev-coverage')
```

Comment:
why prev-coverage for name?

Reply:
This is the previous coverage report produced by clusterfuzz that we are downloading.
We are distinguishing it from the current one we want to generate.